data-serving/scripts/export-data/functions/01-split/app.py

```diff
@@ -0,0 +1,74 @@
+"""Exports MongoDB data to a csv or json file."""
+
+import os
+import json
+
+import boto3
+import pymongo
+
+username = os.environ.get("MONGO_USERNAME")
+password = os.environ.get("MONGO_PASSWORD")
+export_arn = os.environ.get("EXPORT_FUNCTION")
+limit = int(os.environ.get("CHUNK_SIZE"))
+
+
+uri = f"mongodb+srv://{username}:{password}@covid19-map-cluster01.sc7u9.mongodb.net/covid19?retryWrites=true&w=majority"
+client = pymongo.MongoClient(uri)
+db = client.covid19
+cases = db.cases
+
+lambda_client = boto3.client("lambda", region_name="us-east-1")
+s3 = boto3.client("s3")
+
+
+def lambda_handler(event, context):
+    """Global ingestion retrieval function.
+
+    Parameters
+    ----------
+    event: dict, required
+        Input event JSON-as-dict specified by the CloudWatch Event Rule.
+        This must contain a `sourceId` field specifying the canonical epid
+        system source UUID.
+        For more information, see:
+          https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#schedule_event_type
+
+    context: object, required
+        Lambda Context runtime methods and attributes.
+        For more information, see:
+          https://docs.aws.amazon.com/lambda/latest/dg/python-context-object.html
+
+    Returns
+    ------
+    JSON object containing the bucket and key at which the retrieved data was
+    uploaded to S3. For more information on return types, see:
+      https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html
+    """
+
+    num_cases = cases.count_documents({})
+    print(f"There are {num_cases} total cases.")
+    with open("fields.txt", "r") as f:
+        field_names = ",".join(f.read().split("\n"))
+    num_chunks = num_cases // limit
```

Comment:
Yikes good catch thanks.

Reply:
Just gave this a quick look and it doesn't look like it would throw errors in this circumstance actually. Could you give me a quick example of what you're thinking of? Test results for reference:
![image](https://user-images.githubusercontent.com/40274075/109517214-034ce500-7a77-11eb-9940-fc07abc30372.png)